airflow/cli/commands/config_command.py

```diff
@@ -17,11 +17,54 @@
 """Config sub-commands"""
 import io
 
-from airflow.configuration import conf
+from configupdater import ConfigUpdater, NoSectionError
+
+from airflow.configuration import AIRFLOW_CONFIG, AirflowConfigException, conf
 
 
 def show_config(args):
     """Show current application configuration"""
     with io.StringIO() as output:
-        conf.write(output)
+        if args.section:
+            section = args.section
+            if conf.has_section(section):
+                conf._write_section(  # pylint: disable=protected-access
+                    fp=output,
+                    section_name=section,
+                    section_items=conf.getsection(section).items(),
+                    delimiter=" = "
+                )
+            else:
+                raise AirflowConfigException(f"No section '{section}'")
+        else:
+            conf.write(output)
         print(output.getvalue())
+
+
+def set_config_option(args):
+    """Set option in config"""
+    print(f"Making change in {AIRFLOW_CONFIG}")
+
+    if args.executor:
+        section, key, value = "core", "executor", args.executor
+    elif args.dagdir:
+        section, key, value = "core", "dags_folders", args.dagdir
```

Comment:
Why do we need any special options? I think we need to have only `args.section, args.option, args.value`.  No other conditions are needed. This will allow parameters to be positional than making them easier to use.

Reply:
My point was to simplify "most common" options. If you don't think it's a good idea I can remove those.

Reply:
```
airflow ocnfig set core executor MyAwesomeScheduler
```
vs
```
airflow config set --executor MyAwesomeExecutor
airflow config set --section core --option executor --value MyAwesomeExecutor
```
If we simplify, all options will be short enough that they will not require simplification